<launch>
<node
	pkg="robotx_vision"
	type="object_detector"
	name="object_detector"
	required="true"
	output="screen"
>
<param name="subscribed_image_topic" value="my_image"/>
<param name="frame_id" value="my_camera"/>
<param name="output_topic_name" value="my_object"/>
</node>
</launch>
